src/xr/xr-plane-detection.js

```diff
@@ -74,9 +74,12 @@ class XrPlaneDetection extends EventHandler {
      */
 
     _onSessionEnd() {
-        for (let i = 0; i < this._planes.length; i++) {
-            this._planes[i].destroy();
+        if (this._planes) {
```

Comment:
_planes can be null: https://github.com/playcanvas/engine/blob/e76b5faa9c5f34ba9a0dbde424596aaa2cfa6fd8/src/xr/xr-plane-detection.js#L35